### ISSUE TITLE
Analytics (update): <fill in description> 

### ISSUE THREAD

**Issue by signechambers1** [chanzuckerberg/single-cell-data-portal#4534]
### Overview

**Tool**: Gene Expression
Feature: Filters

**Description**: Users can now use the Dataset, Sex, Disease and Ethnicity filters prior to selecting a Tissue or Gene. We'd like to know:
1. Are more visitors filtering before getting to a heat map or after?
2. Is there a specific filter that is more useful to use before vs. after or after vs. before getting to a heat map?

### Updates needed

- Requested change:
~ * includes all of the filters, DATASET, DISEASE, ETHNICITY, SEX
FILTER_SELECT_* ~ to fire when a user selects a filter and there is no heat map loaded yet
FILTER_SELECT_*_BEFORE_HEATMAP  ~ to fire when a user gets to a loaded heat map and a filter is already selected
FILTER_SELECT_*_AFTER_HEATMAP  ~ to fire when a user gets to a loaded heat map and then selects a filter

Checklist

- [ ] Once event updates are deployed to Staging ping @ainfeld to check analytics
- [ ] Verify events are showing up in the Staging space in Plausible correctly
- [ ] Once event updates are deployed to Prod ping @ainfeld
- [ ] Confirm Plausible events are still showing up in the Prod space in Plausible correctly

#### People to cc

**PM**: @signechambers1 
**DS**: @ainfeld

Privacy policy: https://cellxgene.cziscience.com/privacy

**Comment by signechambers1** [comment]
@ashin-czi I'd like to send the names of the filters too please!

Not sure I understand the renaming question, I think any filter selection will be either BEFORE_HEATMAP or AFTER_HEATMAP, so there is no need for a NO_HEATMAP option?

**Comment by tihuan** [comment]
@signechambers1 @dsadgat I might be able to work on this on Thursday if SEO work goes well this week. Will keep you posted!

**Comment by tihuan** [comment]
@signechambers1 not sure who added this, but in `Updates needed` section in the ticket there are three events!

1. FILTER_SELECT_* ~ to fire when a user selects a filter and there is no heat map loaded yet
1. FILTER_SELECT_*_BEFORE_HEATMAP ~ to fire when a user gets to a loaded heat map and a filter is already selected
1. FILTER_SELECT_*_AFTER_HEATMAP ~ to fire when a user gets to a loaded heat map and then selects a filter

It feels like the intention is to take share link into account in the #2 event, so the scenarios below can distinguish all three different filters?

1. User lands on GE
2. User selects filter A -> `FILTER_SELECT_A` fires
3. User selects a gene -> heatmap now renders
4. User selects filter B -> `FILTER_SELECT_B_AFTER_HEATMAP` fires
5. User shares link with UserB
6. UserB uses the share link and lands on GE with filter A and B selected already and heatmap loaded -> `FILTER_SELECT_A_BEFORE_HEATMAP` AND `FILTER_SELECT_B_BEFORE_HEATMAP` fire

Is that right?

**Comment by signechambers1** [comment]
@tihuan you are right FILTER_SELECT_* and FILTER_SELECT_*_BEFORE_HEATMAP feel redundant. @ainfeld i think we might just be able to use one of them, do you agree?

**Comment by tihuan** [comment]
@signechambers1 @ainfeld and I sync'd offline with the following notes:

1. `FILTER_SELECT_*` is to capture user's filter selection BEFORE a heatmap is loaded
2. `FILTER_SELECT_*_BEFORE_HEATMAP` is to capture filters selected AT THE TIME a heatmap is loaded
3. `FILTER_SELECT_*_AFTER_HEATMAP` is to capture additional filters selected AFTER a heatmap is loaded

**Comment by signechambers1** [comment]
Discussed with @ainfeld and @tihuan, we are going to deprioritize this now and reprioritize later if we want to. Product questions would be better served by qualitative research right now.

**Comment by tihuan** [comment]
@signechambers1 I moved this back to "Disorder" and removed the "P1" label!

**Comment by tihuan** [comment]
For completeness: Signe and Amanda don’t see practical business value to track extra analytics for now, so we decided not to spend eng resource on this until we see stronger needs!
